packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.base.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+import { mergeStyles, FocusZone } from 'office-ui-fabric-react';
+import { IProcessedStyleSet, DefaultPalette, IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { IBenchmarkBarChartProps, IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle } from './BenchmarkBarChart.types';
+
+const getClassNames = classNamesFunction<IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle>();
+
+export class BenchmarkBarChartBase extends React.Component<IBenchmarkBarChartProps> {
+  private _classNames: IProcessedStyleSet<IBenchmarkBarChartStyle>;
+
+  constructor(props: IBenchmarkBarChartProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const { chartTitle, benchmarkLabel, data, benchmarkData, totalData } = this.props.data;
+
+    const dataRatio = Math.round((data / totalData) * 100);
+    const benchmarkRatio = Math.round(((benchmarkData ? benchmarkData : 0) / totalData) * 100);
+
+    let percentageContent = data + '/' + totalData;
+    if (this.props.isPercentage) {
+      percentageContent = dataRatio + '%';
+    }
+
+    let benchmarkAriaLabel = benchmarkLabel || '';
+    benchmarkAriaLabel += benchmarkData;
+
+    const benchmarkStyles: IStyle = {
+      marginLeft: 'calc(' + benchmarkRatio + '% - 8px)'
+    };
+
+    const barChartColor = this.props.color || DefaultPalette.purple;
+    const barHeight = this.props.height || 10;
+
+    const { width, benchmarkColor, styles } = this.props;
+    this._classNames = getClassNames(styles!, {
+      width: width,
+      benchmarkColor: benchmarkColor
+    });
+
+    return (
+      <div className={this._classNames.root}>
+        <div className={this._classNames.chartHeader}>
+          <FocusZone>
+            <div className={this._classNames.chartTitle}>
+              <strong data-is-focusable={true}>{chartTitle}</strong>
+            </div>
+          </FocusZone>
+          <FocusZone>
```

Comment:
I think you should be able to put the percentage class name on the focus zone and get rid of a div

Reply:
move percentage classname to focusZone and remove div